dockerfiles/Dockerfile.browser

```diff
@@ -45,7 +45,7 @@ RUN find . -name package.json -type f -mindepth 2 -maxdepth 3 ! -path "./node_mo
 RUN rm -fr **/*/node_modules/
 
 # The maze-runner browser tests
-FROM 855461928731.dkr.ecr.us-west-1.amazonaws.com/maze-runner-releases:latest-v6-cli as browser-maze-runner
+FROM 855461928731.dkr.ecr.us-west-1.amazonaws.com/maze-runner:integration-library-updates-cli as browser-maze-runner
```

Comment:
Oh, remove to update this for maze v7 once released, of course.

Reply:
worth a separate ticket, or a TODO in the code?

Reply:
It'll need changing before this PR is merged.